cmd/list.go

```diff
@@ -63,12 +63,15 @@ func listCmd() *cobra.Command {
 			verbosity, err := cmd.Flags().GetCount("verbose")
 			cobra.CheckErr(err)
 
-			backend := db.NewBackend().
-				Debug(verbosity >= minDebugLevel).
-				WithDatabaseFile(filepath.Join(viper.GetString("cache_dir"), db.DatabaseFile)).
-				WithLogger(utils.NewLogger("list"))
-
-			if err := backend.InitClient(); err != nil {
+			opts := &options.Options{
+				CacheDir: viper.GetString("cache_dir"),
+				Logger:   utils.NewLogger("list"),
+			}
+			backend, err := db.NewBackend(
+				db.Debug(verbosity >= minDebugLevel),
+				db.WithDatabaseFile("bomctl.db"),
+				db.WithOptions(opts))
```

Comment:
```suggestion
			backend, err := db.NewBackend(
				db.Debug(verbosity >= minDebugLevel),
				db.WithDatabaseFile(filepath.Join(viper.GetString("cache_dir"), db.DatabaseFile)),
				db.WithOptions(options.New(
					options.WithCacheDir(viper.GetString("cache_dir")),
					options.WithConfigFile(cfgFile),
					options.WithDebug(verbosity >= minDebugLevel),
					options.WithLogger(utils.NewLogger("")),
				)),
			)
```

Reply:
Slight modification to this that we chatted about, marking it as resolved. 

For others: proposed line 71 used `cfgFile` which doesn't exist in this context, swapped for `viper.ConfigFileUsed()`